### PR TITLE
Fixed linking bug, Index available on every page!

### PR DESCRIPTION
**Pull request by TheDarkLord247**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,230 +1,117 @@
...
-            </p>
-            <p>
-              While fully-connected neural networks look at data as a whole,
-              CNNs look at data <em>one feature at a time</em>. </p>
-            <p>Before we move on, check out <a className="link-to-slides" href="https://teachla.uclaacm.com/classes/ml/10-cnn" target="_blank" rel="noopener noreferrer">these slides</a> to learn about how filters work.</p>
+  const path = window.location.pathname;
+  var pageno = path.substring(1);
+  if (pageno === "") {
+    pageno = "0";
+  }
```
Comment (mattxwang) [inline src/App.js]: Could we cast `pageno` to a number here? Especially as the `page` state is also a number.

File: src/App.js [hunk 2]
```diff
@@ -1,230 +1,117 @@
...
-          <p> But a majority of the images taken today are in color. </p>
-          <p> These images have multiple matrices stacked on top of each other with each matrix containing pixel values for a specific color.
-          In an <em> RGB </em> image, for example, there are 3 matrices - red, green, and blue - stacked on top of each other.
-          Thus, we say that <em> an RGB image has a depth of 3</em>.
-          The topmost channel contains the red intensity values of each pixel, followed by the middle matrix that stores information of the green intensities of each pixel and finally, the last channel that stores information about the blue intensities of each pixel.
-          </p>
-          <p> With three channels instead of one, convolution with an RGB image looks like this: </p>
-          <img src={rgbGIF} alt="RGB filter animation" id="filterGIF" style={{width: '400px'}}/>
+  function RenderButtons(props) {
+    const curPage = props.curPage;
```
Comment (mattxwang) [inline src/App.js]: if `curPage` is supposed to be a number, we can `parseInt`/`number` it here!

File: src/App.js [hunk 3]
```diff
@@ -1,230 +1,117 @@
...
+          <Link to="/3" className="navButton" onClick={() => setPage(3)}>
+            Page 3
+          </Link>
+          <Link to="/4" className="navButton" onClick={() => setPage(4)}>
+            Page 4
+          </Link>
+          <Link to="/5" className="navButton" onClick={() => setPage(5)}>
+            Page 5
+          </Link>
         </div>
```
Comment (mattxwang) [inline src/App.js]: If you'd like, you could generate this with a `.map` over `1...5` - should be less code dup :)

**Review by mattxwang — COMMENTED** [review]
some minor code comments 😊 

I can take a more in-depth view if needed, but also feel free to pair review!

File: src/App.js (hunk 3 above)
Comment (mattxwang) [reply src/App.js]: I think this is the only relevant review now!

**Review by mayaraman19 — APPROVED** [review]

**Pull request merged**